lib/ansible/modules/cloud/azure/azure_rm_loganalyticsworkspace.py

```diff
@@ -116,24 +117,24 @@
 management_groups:
     description:
         - List of management groups connected to the workspace.
```

Comment:
If this data is returned as a dict, we should change this line as well:

```suggestion
        - Management groups connected to the workspace.
```